storage/storage.go

```diff
@@ -40,9 +40,15 @@ type ServiceStore interface {
 	BootstrapTokenStore
 }
 
-// PushStore stores and retrieves APNs push-related data.
+// PushStore retrieves APNs push-related data.
 type PushStore interface {
-	RetrievePushInfo(context.Context, []string) (map[string]*mdm.Push, error)
+	// RetrievePushInfo retrieves push data for the given ids.
+	//
+	// If an ID does not exist or is not enrolled properly then
+	// implementations should silently skip returning any push data for
+	// them. It is up to the caller to discern any missing IDs from the
```

Comment:
How would one discern this is the case? Does this make it more obvious or less obvious what is going in from the consumer perspective?

Reply:
The caller would need to compare the input set of IDs to the returned (output) set of IDs to see what's missing or different. This is how the MySQL backend works, fwiw.